src/Client.php

```diff
@@ -401,6 +402,33 @@ public function send($method, $payload, $path, $params = [], array $headers = []
             return null;
         }
 
+        $responseParsers = [
+            'application/json' => [$this, 'parseJson'],
+            'application/pdf' => [$this, 'parsePdf'],
+        ];
+
+        if (preg_match('/^(\w+\/\w+).*/i', $response->getHeaderLine('Content-Type'), $matches)) {
```

Comment:
(facepalm) some days ago I noticed what `\w` contains `_`, but do not noticed it also contains digits :( I often saw such pattern `[\w\d]` and did not thought to check that is wrong :D